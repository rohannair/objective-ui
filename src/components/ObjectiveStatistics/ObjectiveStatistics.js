import React, { PropTypes } from 'react'
import styled from 'styled-components'

const ObjectiveStatistics = styled.div`
  background: #fff;
  padding: 30px;
`

export default ObjectiveStatistics
