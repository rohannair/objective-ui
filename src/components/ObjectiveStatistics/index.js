import ObjectiveStatistics from './ObjectiveStatistics'
export default ObjectiveStatistics
