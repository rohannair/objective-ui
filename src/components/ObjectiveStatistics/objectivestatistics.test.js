import React from 'react'

import ObjectiveStatistics from './index.js'

describe('Test suite for ObjectiveStatistics component', () => {
  it('ObjectiveStatistics should exist', () => {
    let wrapper = shallow(<ObjectiveStatistics/>)
    expect(wrapper).toMatchSnapshot()
  })
})

