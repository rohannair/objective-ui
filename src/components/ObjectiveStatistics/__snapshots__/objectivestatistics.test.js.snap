exports[`Test suite for ObjectiveStatistics component ObjectiveStatistics should exist 1`] = `
<div
  className="GjDTE" />
`;
